### PR TITLE
Add num days to task cleanup

### PR DESCRIPTION
**Pull request by onobc**

### CODE REVIEW

**Comment by onobc** [comment]
Will add tests in subsequent commit. Want to get this PR in so that the UI side of the feature can progress.

**Comment by klopfdreh** [comment]
Thanks a lot!

File: spring-cloud-dataflow-aggregate-task/src/main/java/org/springframework/cloud/dataflow/aggregate/task/impl/AggregateDataFlowTaskExecutionQueryDao.java [hunk 1]
```diff
@@ -86,75 +89,68 @@ public class AggregateDataFlowTaskExecutionQueryDao implements DataflowTaskExecu
...
+	private static final String GET_EXECUTION_BY_NAME_COMPLETED_BEFORE_END_TIME = GET_EXECUTIONS +
+			" where TASK_NAME = :taskName AND END_TIME IS NOT NULL AND END_TIME < :endTime";
+
+	private static final String GET_EXECUTIONS_COMPLETED_BEFORE_END_TIME = GET_EXECUTIONS +
+			" where END_TIME IS NOT NULL AND END_TIME < :endTime";
 
 	private static final String TASK_EXECUTION_COUNT = "SELECT COUNT(*) FROM "
 			+ "AGGREGATE_TASK_EXECUTION ";
 
+	private static final String TASK_EXECUTION_COUNT_BEFORE_END_TIME = "SELECT COUNT(*) FROM "
```
Comment (klopfdreh) [inline spring-cloud-dataflow-aggregate-task/src/main/java/org/springframework/cloud/dataflow/aggregate/task/impl/AggregateDataFlowTaskExecutionQueryDao.java:133]: I guess `TASK_EXECUTION_COUNT_BEFORE_END_TIME` is not used anymore. So we can remove it.

**Review by klopfdreh — CHANGES_REQUESTED** [review]

**Comment by klopfdreh** [comment]
Only a small nitpick - everything else looks good to me. Thanks again! 😃

**Pull request merged**